pyproject.toml

```diff
@@ -50,7 +50,7 @@ inline-quotes = "double"
 # TODO: should mypy be pre-commit?
 [tool.mypy]
 python_version = "3.10"
-packages = ["pm_tb_data"]
+packages = ["pm_tb_data", "tests"]
```

Comment:
Does including "tests" as a package like this conflict with similar inclusion of "tests" in other modules, e.g. pm_icecon ?

Reply:
Hasn't been a problem on my end. I think this list of packages is relative to the `pyproject.toml`.